### ISSUE TITLE
VSync always active on Linux/Nvidia setup

### ISSUE THREAD

**Issue by v1993** [yuzu-emu/yuzu#6247]
Regardless of emulator setting, VSync is always on (game being emulated is irrelevant). This can be seen by using `__GL_SHOW_GRAPHICS_OSD=1` environment variable, which enables OSD in top left corner. Naturally, one expects the emulator setting to affect syncing behavior.

Workaround: run yuzu via [libstrangle](https://gitlab.com/torkel104/libstrangle) as `strangle -v 0 yuzu`, forcing vsync off. Nvidia's own `__GL_SYNC_TO_VBLANK` variable does not have any effect.

Tested on: KUbuntu 20.10, Nvidia driver 460.73.01, yuzu commit 7412f314e452c90783511102ab7d53d452bf74c2.

**Comment by FernandoS27** [comment]
yuzu's VSync option does not affect your hardware's vSync option. It's actually an option to split Guest's Presentation and do it in a sepparate thread. In theory it's just forcing a the guest game to vSync towards your setup. The Host vSync is something you must configure.

Sorry for the confussion but yeah. Emulators got 2 vSyncs: Guest's (the mulated system's) and Host's(your system). The option works on Guest's.

**Comment by v1993** [comment]
Thanks for explaining what this setting does. In such case, is there any possible benefit from having guest vsync turned off?

The thing is that I have VSync off in Nvidia's control panel (or whatever it's called on Linux) and even with `__GL_SYNC_TO_VBLANK=0` VSync stays on, meaning that yuzu somehow specifically enables it (which libstrangle prevents from happening when used). To contrast, `glxgears` start with VSync off in my setup.

**Comment by FernandoS27** [comment]
> Thanks for explaining what this setting does. In such case, is there any possible benefit from having guest vsync turned off?
> 
> The thing is that I have VSync off in Nvidia's control panel (or whatever it's called on Linux) and even with `__GL_SYNC_TO_VBLANK=0` VSync stays on, meaning that yuzu somehow specifically enables it (which libstrangle prevents from happening when used). To contrast, `glxgears` start with VSync off in my setup.

I can look into that, it shouldn't happen. The advantage is that we don't block the "emulated GPU" thread for presenting the frame in the host,we use a different thread for that.

Now if that setting is off or on and you still have host vSync set then there must be a different issue. Does this happen on Vulkan as well?

**Comment by v1993** [comment]
Yes, host VSync is stuck on both when using OpenGL and Vulkan. Workaround with libstrangle also fixes it for both APIs.
